src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -71,6 +72,16 @@ def sjs_submit(host, port, args, data, retry=None):
             except MaxRetriesExceededError:
                 raise Exception('No slots available in Spark JobServer.\n'
                                 'Details = {}'.format(response.text))
+        elif error['result'] == 'context geoprocessing not found':
+            geop_context_url = '{}/contexts?reset=reboot'.format(base_url)
+            context_response = requests.put(geop_context_url)
```

Comment:
Consider renaming `geop_context_url` to `reboot_sjs_url` or similar to make it more apparent what the code is trying to do here.